app/services/live.js

```diff
@@ -165,4 +193,30 @@ export default class LiveService extends Service {
       await this.hmsActions.detachVideo(presenterTrackId, this.videoEl);
     }
   }
+
+  async removePeer(peerId) {
```

Comment:
removePeer() function is not checking the status code of the response. This means that the function will not be able to handle errors that are returned from the server.

To fix this bug, you need to add a status code check to the removePeer() function. You can do this by adding the following line to the function:

if (response.status !== 200) {
  console.error('The error code is: ', response.status);
  this.toast.error('Something went wrong!', 'error!', TOAST_OPTIONS);
}
This code will now check the status code of the response. If the status code is not 200, the function will display an error message.

The following code shows the corrected code:

async removePeer(peerId) {
    const roomId = this.hmsStore?.getState()?.room?.id;

    const reason = 'For doing something wrong!';
    try {
      const response = await fetch(
        `${ENV.BASE_API_URL}/events/${roomId}/peers/kickout`,
        {
          ...PATCH_API_CONFIGS,
          body: JSON.stringify({
            peerId: peerId,
            reason: reason,
          }),
        }
      );

      if (response.status !== 200) {
        console.error('The error code is: ', response.status);
        this.toast.error('Something went wrong!', 'error!', TOAST_OPTIONS);
      } else {
        const data = await response.json();
        if (data) {
          this.toast.success(data?.message, 'Success!', TOAST_OPTIONS);
        }
      }
    } catch (err) {
      console.error('The error is: ', err);
      this.toast.error('Something went wrong!', 'error!', TOAST_OPTIONS);
    }
  }